### DISCUSSION TITLE
WebView2 and Hololens 2 Windows 11 Update

### DISCUSSION THREAD

**Discussion by Victor-x1** [microsoft/MixedRealityToolkit-Unity#11722] [Ideas]
WebView2 is now available for "Hololens 2 with the Windows 11 Update".  It's currently only available with the Mixed Reality Feature tool. Request to it integrated to MRTK3 MRTKDevTemplate sometime in the far future.
 
 Hololens 2 Windows 11 Update
 https://blogs.windows.com/windowsexperience/2023/04/13/microsoft-brings-windows-11-to-hololens-2/
 
 WebView2 in HoloLens 2
 https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/hololens2

**Comment by AMollis** [comment]
@Victor-x1 , I've converted this to a discussion topic, as it's not an issue will be considering for MRTK3 packages..

I'll point the WebView2 team to this topic, so they can consider adding better support for MRTK3.  However, consideration for this is outside the scope of this project (https://github.com/microsoft/MixedRealityToolkit-Unity)
